packaging/googet/google-osconfig-agent.goospec

```diff
@@ -30,7 +30,6 @@
       ]
   }],
   "build": {
-    "linux": "/bin/bash",
-    "linuxArgs": ["-c", "GOOS=windows /tmp/go/bin/go build -ldflags='-X main.version={{.version}}' -mod=readonly -o google_osconfig_agent.exe"]
```

Comment:
This shouldn't need to be changed as long as you goopack line from packaging/build_goo.sh